src/test/java/it/gov/pagopa/payment/test/fakers/SyncTrxStatusFaker.java

```diff
@@ -0,0 +1,39 @@
+package it.gov.pagopa.payment.test.fakers;
+
+import it.gov.pagopa.payment.constants.PaymentConstants;
+import it.gov.pagopa.payment.dto.qrcode.SyncTrxStatusDTO;
+import it.gov.pagopa.payment.utils.RewardConstants;
+import it.gov.pagopa.payment.utils.Utils;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+
+public class SyncTrxStatusFaker {
+
+    public static SyncTrxStatusDTO mockInstance(Integer bias){
+        return mockInstanceBuilder(bias).build();
+    }
+    public static SyncTrxStatusDTO.SyncTrxStatusDTOBuilder mockInstanceBuilder(Integer bias){
+
+        return SyncTrxStatusDTO.builder()
```

Comment:
Always due to the presence of the mapper's test, we could re-use the TransactionInProgress faker to create a valid instance of that object, then we could use the mapper to transform it into a SyncTrxStatus

This way, if there changes on the valid configuration, we should update just the TransactionInProgress faker